packages/amplify-graphql-conversation-transformer/src/transformer-steps/conversation-resolver-generator.ts

```diff
@@ -48,14 +48,22 @@ export class ConversationResolverGenerator {
     const { functionDataSourceId, referencedFunction } = this.setupFunctionDataSource(directive, functionStack, capitalizedFieldName);
 
     this.createAssistantResponseResolver(ctx, directive, capitalizedFieldName);
-    this.createAssistantResponseSubscriptionResolver(ctx, capitalizedFieldName);
+    this.createAssistantResponseSubscriptionResolver(ctx, directive, capitalizedFieldName);
 
     const functionDataSource = this.addLambdaDataSource(ctx, functionDataSourceId, referencedFunction, capitalizedFieldName);
     const invokeLambdaFunction = invokeLambdaMappingTemplate(directive, ctx);
 
     this.setupMessageTableIndex(ctx, directive);
 
-    this.createConversationPipelineResolver(ctx, parentName, fieldName, capitalizedFieldName, functionDataSource, invokeLambdaFunction);
+    this.createConversationPipelineResolver(
```

Comment:
nit: recommend an options object rather than 7 params

Reply:
What I have in mind for https://github.com/aws-amplify/amplify-category-api/pull/2897#discussion_r1777331065 will include cleaning this up.